mttl/models/utils.py

```diff
@@ -447,11 +454,12 @@ def model_loader_helper(
         )
     elif "phi-2" == model_name:
         # local phi-2 version. use `microsoft/phi-2 for the official hf version`
-        logger.info(
-            f"Loading phi-2 model from {os.getenv('PHI_PATH', 'microsoft/phi-2')}"
-        )
+        if "PHI_PATH" not in os.environ:
```

Comment:
If I remember correctly, the logic here was to not break configs using `phi-2`. If there is no `PHI_PATH` we default to huggingface.

Reply:
ok

Reply:
but actually if we default to microsoft/phi-2 and use our phi private library the code will error out with a weird error